Spigot-API-Patches/0231-Add-senderUuid-to-sendMessage.patch

```diff
@@ -0,0 +1,75 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mariell Hoversholm <proximyst@proximyst.com>
+Date: Sat, 10 Oct 2020 23:55:06 +0200
+Subject: [PATCH] Add senderUuid to sendMessage
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 991f757a89d2e917186bb152a2f92bad8b6c2897..f0020af6a9c474dc599ccc663c331088cd94138a 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -5,6 +5,7 @@ import com.destroystokyo.paper.ClientOption; // Paper
+ import com.destroystokyo.paper.Title; // Paper
+ import com.destroystokyo.paper.profile.PlayerProfile; // Paper
+ import java.util.Date; // Paper
++import java.util.UUID; // Paper
+ import org.bukkit.BanEntry; // Paper
+ import org.bukkit.BanList; // Paper
+ import org.bukkit.Bukkit; // Paper
+@@ -1777,6 +1778,56 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     @NotNull
+     <T> T getClientOption(@NotNull ClientOption<T> option);
++
++    /**
++     * Sends this sender a message raw
++     *
++     * @param message Message to be displayed
++     * @param senderUuid The sender of this message; null for a system message
++     */
++    void sendRawMessage(@Nullable UUID senderUuid, @NotNull String message);
++
++    /**
++     * Sends this player a message
++     *
++     * @param message Message to be displayed
++     * @param senderUuid The sender of this message; null for a system message
++     */
++    void sendMessage(@Nullable UUID senderUuid, @NotNull String message);
++
++    /**
++     * Sends this player multiple messages
++     *
++     * @param messages An array of messages to be displayed
++     * @param senderUuid The sender of this message; null for a system message
++     */
++    void sendMessage(@Nullable UUID senderUuid, @NotNull String[] messages);
++
++    /**
++     * Sends the component to the player
++     *
++     * <p>If this player does not support sending full components then
++     * the component will be sent as legacy text.</p>
++     *
++     * @param component the component to send
++     * @param senderUuid The sender of this message; null for a system message
```

Comment:
Super nitpicky but there's inconsistent capitalization after parameter names throughout this and here is one example <3